tests/Common/ComponentsTest.php

```diff
@@ -1182,16 +1177,22 @@ public function testPermissions()
     public function testTokenWithComponentAccess()
     {
         $this->_initEmptyTestBuckets();
-        $accessibleComponents = array("provisioning");
-        $tokenId = $this->_client->createToken(array($this->getTestBucketId(self::STAGE_IN) => "write"), 'test components', null, false, $accessibleComponents);
+
+        $tokenOptions = (new TokenCreateOptions())
```

Comment:
Tady se nějak ztratil ten první parametr s právama na buckety. Je to ok?

Reply:
Ten tam byl k nicemu. Test testuje jen prava na komponenty, ne na buckety. Takze to tam bylo navic.